install/config.sh

```diff
@@ -0,0 +1,11 @@
+cp .env.example .env
+cp -r .rdeploy.example .rdeploy
+cp config/client.js.example config/client.js
+
+mkdir -p data
+mkdir -p data/rctf-postgres
+mkdir -p data/rctf-redis
```

Comment:
There are some (somewhat nitpicky) issues with these lines, but I'll fix in a future PR.